db/upgrade-10.0.0-10.1.0.sql

```diff
@@ -58,6 +58,11 @@ SET STATEMENT sql_mode='NO_AUTO_VALUE_ON_ZERO' FOR
 \! echo "PIDs should be unique in the password table"
 CREATE UNIQUE INDEX IF NOT EXISTS `pid_password_unique` ON password(pid);
 
+\! echo "altering pki_profiles"
+ALTER TABLE pki_profiles
+    ADD COLUMN IF NOT EXISTS `cloud_enabled` int(11) AFTER scep_days_before_renewal,
+    ADD COLUMN IF NOT EXISTS `cloud_service` varchar(255) AFTER cloud_enabled;
+
```

Comment:
Remove from here and add it to ``` db/upgrade-X.X-X.Y.sql ```

Reply:
it's already done in the previous commit.